Resources/Prototypes/Stories/Catalog/uplink_catalog.yml

```
@@ -0,0 +1,9 @@
+- type: listing
+  id: UplinkRadioJammerIndustrial
+  name: Промышленный глушитель связи
+  description: Это устройство является улучшенной версией обычного глушителя связи. Радиус действия увеличен, но и потребление больше. (Замаскирован под %%%%%%%)
+  productEntity: RadioJammerIndustrial
+  cost:
+    Telecrystal: 12
+  categories:
+  - UplinkDisruption
```

```suggestion
- type: listing
  id: UplinkRadioJammerIndustrial
  name: Экспериментальный глушитель связи
  description: Это устройство является улучшенной версией обычного глушителя связи. Радиус действия увеличен, но и потребление больше.
  productEntity: RadioJammerIndustrial
  cost:
    Telecrystal: 6
  categories:
  - UplinkDisruption
```

За столько ТК никто это не купит.

---

Обычная 6 стоит, какой смысл?

---

4 стоит обычная